code/modules/farming/bottle_kit.dm

```diff
@@ -45,3 +51,38 @@
 
 	fake_glass_name = choice
 	glass_colour = printing_choice
+
+/obj/item/reagent_containers/glass/bottle/brewing_bottle
+	name = "brewer's bottle"
+	desc = "A bottle with a special cork. When sealed, it is valuable commodity for trade."
+	icon =  'icons/obj/bottle.dmi'
+	icon_state = "brew_bottle"
+	volume = 50
+	can_label_bottle = FALSE
+
+	// var/glass_name
+	// var/glass_desc
+
+/obj/item/reagent_containers/glass/bottle/brewing_bottle/examine(mob/user)
+	. = ..()
+	if(sellprice)
+		. += span_notice("The bottle is sealed and can be sold on the market.")
+
+/obj/item/reagent_containers/glass/bottle/brewing_bottle/toggle_cork(mob/user)
+	if(closed && sellprice && reagents?.total_volume)
+		to_chat(user, span_boldwarning("You start to unseal [src], ruining its economic value."))
```

Comment:
no need to say that you are "ruining" it . i think its already implied that drinking it and unsealing will removenits value